api/graphql/reservation_units/reservation_unit_types.py

```diff
@@ -83,9 +84,15 @@ def resolve_keyword_groups(self, info):
 
 
 class PurposeType(AuthNode, PrimaryKeyObjectType):
+    permission_classes = (
+        (PurposePermission,) if not settings.TMP_PERMISSIONS_DISABLED else (AllowAny,)
+    )
+
     class Meta:
         model = Purpose
-        fields = ("id", "name", "pk")
+        fields = ("id", "pk", "name_fi", "name_en", "name_sv")
```

Comment:
I was thinking more about this, and I think it might be okay to leave `name` here for now, because removing it will break the existing API and reservation unit / purpose tests.

We could do a separate PR where we remove the non-language specific fields from all types.

The other option would be to fix the broken reservation unit and purpose tests that already rely on `name` being there.

Reply:
I'll fix the tests and let's change the api since that's the way we want it to be.